lib/battle/Unit.h

```diff
@@ -41,6 +41,27 @@ namespace BattlePhases
 	};
 }
 
+// Healed HP (also drained life) and resurrected units info
+struct HealInfo
+{
+	HealInfo()
+		: healedHealthPoints(0), resurrectedCount(0)
+	{ }
+	HealInfo(int64_t healedHP, int32_t resurrected)
+		: healedHealthPoints(healedHP), resurrectedCount(resurrected)
+	{ }
+
+	int64_t healedHealthPoints;
+	int32_t resurrectedCount;
+
+	HealInfo& operator +=(const HealInfo& other)
```

Comment:
by our coding style it should be:
```suggestion
	HealInfo & operator+=(const HealInfo & other)
```